morebits.js

```diff
@@ -321,7 +325,16 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 			if (data.label) {
 				label = node.appendChild(document.createElement('label'));
 				label.setAttribute('for', id);
-				label.appendChild(document.createTextNode(data.label));
+				if (!Array.isArray(data.label)) {
+					data.label = [ data.label ];
+				}
+				for (i = 0; i < data.label.length; ++i) {
+					if (typeof data.label[i] === 'string') {
+						label.appendChild(document.createTextNode(data.label[i]));
+					} else if (data.label[i] instanceof Element) {
+						label.appendChild(data.label[i]);
+					}
+				}
```

Comment:
I think this can be written as `$(label).append(data.label);` and there's no reason morebits should continue to pretend jquery doesn't exist. 

In any case, this is repeated 7 times in this PR itself and I think once in Morebits.status, so should be pulled out as a function.

Reply:
> I think this can be written as $(label).append(data.label);

Oh, this does mean that if label text is based on user input it would need to be sanitised first to avoid XSS issue - which I'd prefer be done on the client script side - which means a breaking change.

Reply:
I'm not entirely sure what you're suggesting with this comment, but

>done on the client script side

I'd suppose so, yes.

Reply:
Oh, I think I get it now.  You're saying that `.append` would allow for xss, which is a reason against doing the jQuery route, but that in theory that should be dealt with by whatever is using the library, right?  And that's a breaking change?

Reply:
Yes, but it (the XSS issue) may not be a worthwhile consideration as I doubt anyone's using quickform that innovatively!